modules/ROOT/pages/provisioning-proxmoxve.adoc

```diff
@@ -0,0 +1,192 @@
+= Provisioning Fedora CoreOS on Proxmox VE
+
+== Prerequisites
+
+Before provisioning an FCOS machine on Proxmox VE, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a https://www.proxmox.com[Proxmox Virtual Environment] (VE) cluster or standalone server with administrative privileges to create VMs and storage configurations. All of the commands will be run as the `root` user on the Proxmox VE host.
+
+== Setting up Proxmox VE Storage
+
+Before provisioning Fedora CoreOS VMs, you need to create a dedicated storage location for FCOS images and Ignition configuration snippets.
+
+=== Creating the Storage Directory
+
+First, create a directory structure for Fedora CoreOS resources:
+
+[source,bash]
+----
+mkdir -p /var/coreos
+----
+
+=== Adding Storage to Proxmox VE
+
+Add the new directory as a Proxmox VE storage location that can hold both VM images and configuration snippets:
+
+[source,bash]
+----
+pvesm add dir coreos --path /var/coreos --content images,snippets
+----
+
+This command creates a new storage location named `coreos` that Proxmox VE can use for storing disk images and configuration snippets (including Ignition files). For more details on Proxmox VE storage configuration, see the https://pve.proxmox.com/wiki/Storage[Proxmox VE Storage documentation].
+
+== Downloading Fedora CoreOS Images
+
+Fedora CoreOS provides pre-built images specifically optimized for Proxmox VE. You can download these images using either the `coreos-installer` binary or via a container.
+
+=== Fetching the QCOW2 image
+
+Fetch the latest image suitable for your target stream (or https://fedoraproject.org/coreos/download/[download and verify] it from the web).
+
+[source,bash]
+----
+STREAM="stable"
+# as an installed binary:
+coreos-installer download -s $STREAM -p proxmoxve -f qcow2.xz --decompress -C /var/coreos
+# or as a container:
+podman run --pull=always --rm -v "/var/coreos/images:/data" -w /data \
+    quay.io/coreos/coreos-installer:release download -s $STREAM -p proxmoxve -f qcow2.xz --decompress
+----
+
+Both methods will download the latest Fedora CoreOS image for the specified stream (`stable`, `testing`, or `next`) in QCOW2 format, optimized for Proxmox VE. It will store the image in the `/var/coreos/images` directory.
+
+== Preparing Ignition Configuration
+
+Upload your Ignition configuration file to the snippets directory:
+
+[source,bash]
+----
+# Upload your ignition to /var/coreos/config.ign
+scp /path/to/your/config.ign root@proxmoxve-host:/var/coreos/config.ign
```

Comment:
```suggestion
scp /path/to/your/config.ign root@proxmoxve-host:/var/coreos/snippets/config.ign
```
I think the ignition config needs to be copied into the snippets directory to be discovered during VM startup